### DISCUSSION TITLE
How do I get into the docker container to edit my config?

### DISCUSSION THREAD

**Discussion by kroung** [bluenviron/mediamtx#1164] [Questions]
I have the server up and running, and installed on a docker container.

2022/09/21 19:44:13 INF rtsp-simple-server v0.20.0
2022/09/21 19:44:13 INF [RTSP] listener opened on :8554 (TCP), :8000 (UDP/RTP), :8001 (UDP/RTCP)
2022/09/21 19:44:13 INF [RTMP] listener opened on :1935
2022/09/21 19:44:13 INF [HLS] listener opened on :8888

But I cant seem to figure out how to get into editing the config file. Forgive me I am super new to docker. 

When I try to get into bash or sh I get $path error: 

root@vultr:~# docker exec -it e6e63c9fb1e2 bash
OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "bash": executable file not found in $PATH: unknown
root@vultr:~#  starting container process caused: exec: "bash": executable file not found in $PATH: unknown
starting: command not found
root@vultr:~# docker exec -it e6e63c9fb1e2 sh
OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "sh": executable file not found in $PATH: unknown

root@vultr:~# docker run --rm -it --network=host -v $PWD/rtsp-simple-server.yml:/rtsp-simple-server.yml aler9/rtsp-simple-server
docker: Error response from daemon: OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:545: container init caused: rootfs_linux.go:76: mounting "/root/rtsp-simple-server.yml" to rootfs at "/rtsp-simple-server.yml" caused: mount through procfd: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type.
ERRO[0000] error waiting for container: context canceled

Thanks!

**Comment by ShyLionTjmn** [comment]
Are you sure /root/rtsp-simple-server.yml file exists? just checked, it works as should

**Comment by 459737087** [comment]
Hello did you solve it,

**Reply by ShyLionTjmn** [reply]
docker run --rm -it --network=host -v /root/rtsp-simple-server.yml:/rtsp-simple-server.yml aler9/rtsp-simple-server

Simply worked without issue for me.

**Reply by 459737087** [reply]
> --network=host -v /root/rtsp-simple-server.yml:/rtsp-simple-server.yml aler9/rtsp-simple-serv

thank you

[a comment by an automated account is not shown]
